### PR TITLE
remove Bokeh deprecation warnings #1657

### PR DESCRIPTION
**Pull request by vikramnagashoka**
## Description
Replaced the Dash and Cross functions as they are deprecated with Scatter function from the bokeh.models.glyphs module. 

## Checklist

- [x] Follows [official](https://github.com/arviz-devs/arviz/blob/main/CONTRIBUTING.md#pull-request-checklist) PR format
- [x] Code style  correct (follows pylint and black guidelines)
- [ ] Changes are listed in [changelog](https://github.com/arviz-devs/arviz/blob/main/CHANGELOG.md#v0xx-unreleased)

### CODE REVIEW

File: arviz/plots/backends/bokeh/elpdplot.py [hunk 1]
```diff
@@ -163,9 +163,7 @@ def _plot_atomic_elpd(
     marker = _validate_bokeh_marker(plot_kwargs.get("marker"))
     marker_func = getattr(mk, marker)
     sizes = np.ones(len(xdata)) * plot_kwargs.get("s")
-    glyph = marker_func(
-        x="xdata", y="ydata", size="sizes", line_color=plot_kwargs.get("color", "black")
-    )
+    glyph = Scatter(x="xdata", y="ydata", size="sizes", line_color=plot_kwargs.get("color", "black"), marker='cross')
```
Comment (OriolAbril) [inline arviz/plots/backends/bokeh/elpdplot.py:166]: note that while the other instances call `Dash` explicitly, here the `marker_func` is used instead, which allows users to choose the marker to be used. I believe the suggestion will fix this change in funcitonality, marker is already defined correctly, but you should definitely try it out locally

```suggestion
    glyph = Scatter(x="xdata", y="ydata", size="sizes", line_color=plot_kwargs.get("color", "black"), marker=marker)
```

**Review by OriolAbril — CHANGES_REQUESTED** [review]
Thanks for the PR

**Review by vikramnagashoka — COMMENTED** [review]
Updated to use marker variable instead of constant value of 'cross'

**Comment by OriolAbril** [comment]
You have to make sure that all tests pass. Do you need help interpreting the ci errors?

**Comment by vikramnagashoka** [comment]
Yes please. I see that the error is related to pylint. The output from this task says score is 10/10. Also when I navigated to the azure pipeline it said there were other jobs which failed with same error.

**Comment by OriolAbril** [comment]
You have to take a look at the messages at https://dev.azure.com/ArviZ/ArviZ/_build/results?buildId=4342&view=logs&j=af9a1d3a-ff8f-5483-6823-86518c773a9b&t=91d5999d-c3e7-502d-15b0-1abfa51eab6f&l=14, I never really understood the score out of 10 that gets assigned. What's important is to fix all the messages, even if it says 10, if there are error messages, there are things wrong and CI will therefore fail. The errors are related to lines too long (running black should fix this) and to extra code that is no longer needed with the `Scatter` instead of `marker_fun` change

**Comment by OriolAbril** [comment]
It looks like you have modified many unrelated files and now black is complaining. Could it be that you ran black but you have an old version of black installed?

**Comment by vikramnagashoka** [comment]
@OriolAbril not sure what you mean by unrelated files. All the files that I changed had either the Dash or Cross functions which I replaced with Scatter. Can you please let me know what I'm missing here ? 

The black version I'm using is as below. 
_**black, version 21.4b2**_

Can you please help me resolve this ?

**Comment by OriolAbril** [comment]
I must have been looking at a wrong commit or misread the filenames, sorry. CI is currently failing at the black step: https://dev.azure.com/ArviZ/ArviZ/_build/results?buildId=4344&view=logs&j=af9a1d3a-ff8f-5483-6823-86518c773a9b&t=6a1cfc46-f4a4-5e3c-e2fb-18ea29f733bf&l=14. It looks like (https://dev.azure.com/ArviZ/ArviZ/_build/results?buildId=4344&view=logs&j=af9a1d3a-ff8f-5483-6823-86518c773a9b&t=0aaa11b3-1b3c-5ed3-0c54-8fd8e63dd2e8&l=25) you have the same version installed as the one running on CI, so you should run black and commit the changes to fix this. The command to run black on ArviZ is on the PR checklist: https://github.com/arviz-devs/arviz/blob/main/CONTRIBUTING.md#pull-request-checklist

**Comment by vikramnagashoka** [comment]
@OriolAbril thanks for the details. I have pushed changes after running black and now all checks pass. Can you please help me merge the branch ?

**Review by OriolAbril — APPROVED** [review]
Thanks for updating and fixing all the tests, you should add the fixes to the changelog in the "maintenence and fixes" section and we'll merge the PR :smile: 

If you are worried about the PR not being merged eventually I can already guarantee it will be merged

**Comment by OriolAbril** [comment]
Thanks @vikramnagashoka

**Pull request merged**